### PR TITLE
Casmins 4311

### PR DESCRIPTION
**Pull request by mantoanibroadwing**
Updated index.md

### CODE REVIEW

File: troubleshooting/index.md [hunk 1]
```diff
@@ -2,101 +2,82 @@
...
-    * [General Kubernetes Commands for Troubleshooting](./kubernetes/Kubernetes_Troubleshooting_Information.md)
-    * [Kubernetes Log File Locations](./kubernetes/Kubernetes_Log_File_Locations.md)
-    * [Troubleshoot Liveliness or Readiness Probe Failures](./kubernetes/Troubleshoot_Liveliness_Readiness_Probe_Failures.md)
-    * [Troubleshoot Unresponsive kubectl Commands](./kubernetes/Troubleshoot_Unresponsive_kubectl_Commands.md)
-    * [Troubleshoot Kubernetes Node NotReady](./kubernetes/Troubleshoot_Kubernetes_Node_NotReady.md)
+    * [Hardware Discovery](Hardware_Discovery.md)
+    * [SAT/HSM/CAPMC Component Power State Mismatch](https://github.com/Cray-HPE/docs-csm/blob/CASMINS-4311/troubleshooting/known_issues/component_power_state_mismatch.md)
 
+### Troubleshoot Topics
+ * Kubernetes troubleshooting
```
Comment (nrockershousen) [inline troubleshooting/index.md:11]: Change to "Troubleshoot Kubernetes" to be consistent with items below

File: troubleshooting/index.md [hunk 2]
```diff
@@ -2,101 +2,82 @@
...
+      * [Troubleshoot Stale Brokered UAIs](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/UAS_user_and_admin_topics/Troubleshoot_Stale_Brokered_UAIs.md)
+      * [Troubleshoot UAI Stuck in "ContainerCreating"](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/UAS_user_and_admin_topics/Troubleshoot_UAI_Stuck_in_ContainerCreating.md)
+      * [Troubleshoot Duplicate Mount Paths in a UAI](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/UAS_user_and_admin_topics/Troubleshoot_Duplicate_Mount_Paths_in_a_UAI.md)
+      * [Troubleshoot Missing or Incorrect UAI Images](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/UAS_user_and_admin_topics/Troubleshoot_Missing_or_Incorrect_UAI_Images.md)
+     
+      * [Troubleshoot Common Mistakes when Creating a Custom End-User UAI Image](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/UAS_user_and_admin_topics/Troubleshoot_Common_Mistakes_when_Creating_a_Custom_End-User_UAI_Image.md)
+ * Troubleshoot Boot issues
```
Comment (nrockershousen) [inline troubleshooting/index.md:30]: Change "issues" to "Issues"

File: troubleshooting/index.md [hunk 3]
```diff
@@ -2,101 +2,82 @@
...
+      * [Troubleshoot UAN Boot Issues](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/boot_orchestration/Troubleshoot_UAN_Boot_Issues.md)
+      *
```
Comment (nrockershousen) [inline troubleshooting/index.md:32]: remove empty bullet point

File: troubleshooting/index.md [hunk 4]
```diff
@@ -2,101 +2,82 @@
...
+ * Troubleshoot Utility Storage
+      * [Troubleshoot Failure to Get Ceph Health](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/utility_storage/Troubleshoot_Failure_to_Get_Ceph_Health.md)
+      * [Troubleshoot a Down OSD](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/utility_storage/Troubleshoot_a_Down_OSD.md)
+      * [Troubleshoot Ceph OSDs Reporting Full](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/utility_storage/Troubleshoot_Ceph_OSDs_Reporting_Full.md)
+      * [Troubleshoot System Clock Skew](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/utility_storage/Troubleshoot_System_Clock_Skew.md)
+      * [Troubleshoot an Unresponsive S3 Endpoint](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/utility_storage/Troubleshoot_an_Unresponsive_S3_Endpoint.md)
+      * [Troubleshoot Ceph-Mon Processes Stopping and Exceeding Max Restarts](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/utility_storage/Troubleshoot_Ceph-Mon_Processes_Stopping_and_Exceeding_Max_Restarts.md)
+
```
Comment (nrockershousen) [inline troubleshooting/index.md:60]: remove blank line

File: troubleshooting/index.md [hunk 5]
```diff
@@ -2,101 +2,82 @@
...
+ * Troubleshoot Node Management
+      * [Troubleshoot Issues with Redfish Endpoint DiscoveryCheck for Redfish Events from Nodes](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/node_management/Troubleshoot_Issues_with_Redfish_Endpoint_Discovery.md)
+      * [Troubleshoot Interfaces with IP Address Issues](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/node_management/Troubleshoot_Interfaces_with_IP_Address_Issues.md)
+      * [Troubleshoot Loss of Console Connections and Logs on Gigabyte Nodes](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/node_management/Troubleshoot_Loss_of_Console_Connections_and_Logs_on_Gigabyte_Nodes.md)
+ * Troubleshoot Customer Access Network (CAN)
+      * [Troubleshoot CAN Issues](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/network/customer_access_network/Troubleshoot_CAN_Issues.md)
+ * Troubleshoot Dynamic Host Configuration Protocol (DHCP)
+      
+ * Domain Name Service (DNS)
+
```
Comment (nrockershousen) [inline troubleshooting/index.md:72]: remove blank line

File: troubleshooting/index.md [hunk 6]
```diff
@@ -2,101 +2,82 @@
...
+      * [Troubleshoot Failure of RGW Health Check](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/utility_storage/Troubleshoot_RGW_Health_Check_Fail.md)
+ * Troubleshoot Node Management
+      * [Troubleshoot Issues with Redfish Endpoint DiscoveryCheck for Redfish Events from Nodes](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/node_management/Troubleshoot_Issues_with_Redfish_Endpoint_Discovery.md)
+      * [Troubleshoot Interfaces with IP Address Issues](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/node_management/Troubleshoot_Interfaces_with_IP_Address_Issues.md)
+      * [Troubleshoot Loss of Console Connections and Logs on Gigabyte Nodes](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/node_management/Troubleshoot_Loss_of_Console_Connections_and_Logs_on_Gigabyte_Nodes.md)
+ * Troubleshoot Customer Access Network (CAN)
+      * [Troubleshoot CAN Issues](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/network/customer_access_network/Troubleshoot_CAN_Issues.md)
+ * Troubleshoot Dynamic Host Configuration Protocol (DHCP)
+
```
Comment (nrockershousen) [inline troubleshooting/index.md:70]: remove blank line

File: troubleshooting/Hardware_Discovery.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
+### Hardware Discovery
```
Comment (nrockershousen) [inline troubleshooting/Hardware_Discovery.md:1]: - This file should be moved to the "troubleshooting/known_issues" subdirectory
- Change this to a top-level header `# Hardware Discovery`

File: troubleshooting/Hardware_Discovery.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+Known issues related to hardware discovery in a system.
+ * [Air cooled hardware is not getting properly discovered with Aruba leaf switches](known_issues/discovery_aruba_snmp_issue.md)
+ * [HMS Discovery job not creating RedfishEndpoints in Hardware State Manager](known_issues/discovery_job_not_creating_redfish_endpoints.md)
+
+<a name="initrd-not-found"></a>
+### ``error: file  `/boot/grub2/../initrd.img.xz' not found.Press any key to continue...``
```
Comment (nrockershousen) [inline troubleshooting/Hardware_Discovery.md:7]:
```suggestion
## ``error: file  `/boot/grub2/../initrd.img.xz' not found.Press any key to continue...``
```

File: troubleshooting/Hardware_Discovery.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+[    2.570431]  prepare_namespace+0x130/0x166
+[    2.574645]  kernel_init_freeable+0x23f/0x26b
+[    2.579125]  ? rest_init+0xb0/0xb0
+[    2.582623]  kernel_init+0xa/0x110
+[    2.586127]  ret_from_fork+0x22/0x40
+[    2.590747] Kernel Offset: 0x0 from 0xffffffff81000000 (relocation range: 0xffffffff80000000-0xffffffffbfffffff)
+[    2.690969] ---[ end Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(0,0) ]---
+```
+
+#### Fix
```
Comment (nrockershousen) [inline troubleshooting/Hardware_Discovery.md:32]:
```suggestion
### Fix
```

File: troubleshooting/Hardware_Discovery.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+      ncn# pdsh -b -S -w $(grep -oP 'ncn-\w\d+' /etc/hosts | sort -u |  tr -t '\n' ',') '/opt/cray/csm/workarounds/livecd-post-reboot/CASMINST-2689/CASMINST-2689.sh'
+      ```
+
+   1. Remove the Shasta 1.4 install workaround RPM from the NCN.
+
+      ```bash
+      ncn# rpm -e csm-install-workarounds
+      ```
+
+##### Validate
```
Comment (nrockershousen) [inline troubleshooting/Hardware_Discovery.md:58]:
```suggestion
### Validate
```

File: troubleshooting/index.md [hunk 11]
```diff
@@ -2,101 +2,82 @@
...
-    * [Hardware Discovery](#known-issues-hardware-discovery)
-    * [initrd.img.xz not found](#initrd-not-found)
-    * [SAT/HSM/CAPMC Component Power State Mismatch](known_issues/component_power_state_mismatch.md)
- * Kubernetes troubleshooting
-    * [General Kubernetes Commands for Troubleshooting](./kubernetes/Kubernetes_Troubleshooting_Information.md)
-    * [Kubernetes Log File Locations](./kubernetes/Kubernetes_Log_File_Locations.md)
-    * [Troubleshoot Liveliness or Readiness Probe Failures](./kubernetes/Troubleshoot_Liveliness_Readiness_Probe_Failures.md)
-    * [Troubleshoot Unresponsive kubectl Commands](./kubernetes/Troubleshoot_Unresponsive_kubectl_Commands.md)
-    * [Troubleshoot Kubernetes Node NotReady](./kubernetes/Troubleshoot_Kubernetes_Node_NotReady.md)
+    * [Hardware Discovery](Hardware_Discovery.md)
```
Comment (nrockershousen) [inline troubleshooting/index.md:7]: Update the path to "known_issues/Hardware_Discovery.md" when this file is moved.

File: troubleshooting/index.md [hunk 12]
```diff
@@ -2,101 +2,82 @@
 
 This document provides troubleshooting information for services and functionality provided by CSM.
 
-### Topics
+### Known Issues
```
Comment (nrockershousen) [inline troubleshooting/index.md:5]: General comment:

We don't need to include the full URL for links to other markdown files in the same repository.

Comment (mantoanibroadwing) [reply troubleshooting/index.md:5]: Agreed and this was take under consideration was done like this on this document specific just to make sure no broken links

File: troubleshooting/Hardware_Discovery.md (hunk 7 above)
Comment (mantoanibroadwing) [reply troubleshooting/Hardware_Discovery.md:1]: Completed

**Review by mantoanibroadwing — COMMENTED** [review]
Completed change

**Review by mantoanibroadwing — COMMENTED** [review]
Removed line

**Review by phalseth-hpe — CHANGES_REQUESTED** [review]
it looks like there are 3 .idea files in this PR that shouldn't be in here - can those be removed?

**Comment by mantoanibroadwing** [comment]
Need to be open for release

**Pull request closed without merging**